modules/social_features/social_landing_page/css/social_landing_page.section.hero.css

```diff
@@ -34,10 +34,12 @@
     margin: 0 0 5px;
     font-weight: normal;
     font-size: 20px; }
+  .paragraph--hero .cover-small .hero-footer .field--name-field-hero-buttons {
+    margin: 0 -10px; }
   .paragraph--hero .cover-small .hero-footer .field--item {
     display: inline-block; }
-    .paragraph--hero .cover-small .hero-footer .field--item:not(:first-child) .paragraph--button {
-      padding-left: 20px; }
+    .paragraph--hero .cover-small .hero-footer .field--item .paragraph--button {
+      padding: 0 10px; }
```

Comment:
Why don't we just change this to `padding-right: 10px` then your negative margin is not needed.

Reply:
Then content(buttons) will stretch correctly in left and right corner.

Reply:
I don't understand. In my testing, padding-right: 10px is enough. I don't see any negative side effects to this.
I prefer this over needing to add a negative margin.
@Crazylord can you explain please, otherwise perhaps you can review this please

Reply:
If you set padding-right - then buttons will be break to another line earlier. In my case, button will be stick to right side

Reply:
Understood. Thanks for explaining